main.rb

```diff
@@ -0,0 +1,54 @@
+require './app'
+
+class Apps
+  def self.menu
+    puts 'Welcom to the School Library'
```

Comment:
- [ ] OPTIONAL
It will be nice to correct the typo from _welcom_ to _welcome_

Reply:
Hi @rloterh  thank you for your approval, typo corrected